packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -6,22 +6,25 @@ excerpt: >-
 
 # Client-based rate limits
 
-Client-based limiting uses either of these two endpoints:
+ Client-based rate limiting applies to unauthenticated endpoints within Okta that are used during the client application access process. A client being, in this context, one of two types:
 
-* OAuth 2.0 `/authorize`
-* `/login/login.htm`
+* An OAuth 2.0 application, which uses a combination of client ID, IP address, or device identifier for rate-limiting.
```

Comment:
```suggestion
* An OAuth 2.0 application, which uses a combination of client ID, IP address, or device identifier for rate-limiting
```
fragment?

Reply:
I like Maurice's idea of periods after everything. Makes life easier for writers, haha.